web-app/src/pages/Router.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { BrowserRouter, Switch, Route, Redirect } from "react-router-dom";
+import Game from "../components/Game";
+import MainMenu from "../components/MainMenu";
+import About from "./About";
+
+type RouteProps = {
+	urls: { [key: string]: string };
+};
+
+export const RouteConfig = {
+	About: '/about',
+	Game: '/game/:gameId',
+	GameChatChannel: '/game/:gameId/Channel/:channelId/Messages',
```

Comment:
If the "game" path element is in lowercase, shouldn't the "channel" and "messages" elements be as well?

Reply:
And "lab" below.

Reply:
I changed it.